src/main/java/frc/robot/subsystems/SwerveModule.java

```diff
@@ -173,6 +173,16 @@ public void setDesiredState(SwerveModuleState desiredState) {
         desiredSpeedLog.append(desiredState.speedMetersPerSecond);
     }
 
+    public void park(boolean reversed) {
+        if(reversed){
+            //not using calculateSetpoint because these are less than one full rotation
+            turningPIDController.setReference(-Math.PI/4, ControlType.kPosition);  
+        } else{
+            turningPIDController.setReference(Math.PI/4, ControlType.kPosition);
+        }
+        stop();
```

Comment:
This will override your set reference. Put stop() before setting the turningPIDController